### PR TITLE
Rewrite tutorial to incorporate pagination and reformatted code.

### PR DESCRIPTION
**Pull request by conshus**
## Description

Rewrite tutorial to incorporate pagination and reformatted code.

## Review

* [This page](https://nexmo-developer-pr-2618.herokuapp.com/client-sdk/tutorials/in-app-messaging/introduction)

### CODE REVIEW

**Comment by conshus** [comment]
Will definitely need some help with the wording. haha Anything you can do is greatly appreciated. Thanks.

File: _tutorials/en/client-sdk/ip-messaging/show-message-history.md [hunk 1]
```diff
@@ -7,23 +7,70 @@ description: In this step you display any messages already sent as part of this
 
 You want your users to see all the messages in the Conversation. You can achieve this by handling the Conversation’s `getEvents` method (to retrieve messages sent and received before the current session started) and its `text` event (which alerts your application when a user sends a message).
 
+In the case when the number of messages are more than the page size of the request, you can use `getNext()` to receive the next page. More information on `getNext()` can be found in the [documentation](https://developer.nexmo.com/sdk/stitch/javascript/EventsPage.html#getNext). You will call this when the "Load Previous Messages" button is clicked.
```
Comment (tbedford) [inline _tutorials/en/client-sdk/ip-messaging/show-message-history.md:10]: I had a couple of problems with this section - 1) It's not obvious where to paste this function in the code. I think it should go beneath the `loginForm.addEventListener` function, but this is not explicitly stated. 2) `loadMessagesButton` is not defined anywhere - it should possibly be declared below `const loginForm = document.getElementById("login");`. These two issues lead to errors when running the example.

Comment (tbedford) [inline _tutorials/en/client-sdk/ip-messaging/show-message-history.md:10]: "In the case when the number" -> In the case where the number

Comment (tbedford) [inline _tutorials/en/client-sdk/ip-messaging/show-message-history.md:10]: "You will call this when the "Load Previous Messages" button is clicked." -> perhaps better as "This function is called when the **Load Previous Messages** button is clicked."

File: _tutorials/en/client-sdk/ip-messaging/create-ui.md [hunk 2]
```diff
@@ -89,14 +90,21 @@ Create a file named `index.html` in your project directory with the following co
 
   <section id="messages">
     <h1 id="sessionName"></h1>
+
+    <div id="loadSection">
+      <button id="loadMessages">
```
Comment (tbedford) [inline _tutorials/en/client-sdk/ip-messaging/create-ui.md:95]: Minor styling issue: not sure why but the "Load Previous" button looks strangely clipped to me.

File: _tutorials/en/client-sdk/ip-messaging/show-message-history.md (hunk 1 above)
Comment (conshus) [reply _tutorials/en/client-sdk/ip-messaging/show-message-history.md:10]: You're right. I forgot to add some code in a previous step. Going to add it now. Thanks for the catch.

Comment (conshus) [reply _tutorials/en/client-sdk/ip-messaging/show-message-history.md:10]: Got it.

Comment (conshus) [reply _tutorials/en/client-sdk/ip-messaging/show-message-history.md:10]: 👍🏽Thanks.

File: _tutorials/en/client-sdk/ip-messaging/create-ui.md (hunk 2 above)
Comment (conshus) [reply _tutorials/en/client-sdk/ip-messaging/create-ui.md:95]: hmmmm it doesn't look like this for you?
image
Can you post a screenshot? Thanks.

Comment (tbedford) [reply _tutorials/en/client-sdk/ip-messaging/create-ui.md:95]: image

**Comment by tbedford** [comment]
@conshus not sure what happened while I was away but I notice there are now a number of merge conflicts to sort. Perhaps @fabianrbz reorganized things?

File: _tutorials/en/client-sdk/ip-messaging/create-ui.md (hunk 2 above)
Comment (conshus) [reply _tutorials/en/client-sdk/ip-messaging/create-ui.md:95]: What browser are you using? 

I also noticed that there's no number after "Showing" or a date. And it's below the button.

Are things zoomed in?

**Comment by conshus** [comment]
> @conshus not sure what happened while I was away but I notice there are now a number of merge conflicts to sort. Perhaps @fabianrbz reorganized things?

Yeah, I have no clue. Something must have changed in the main branch.

File: _tutorials/en/client-sdk/ip-messaging/create-ui.md (hunk 2 above)
Comment (tbedford) [reply _tutorials/en/client-sdk/ip-messaging/create-ui.md:95]: Chrome

**Comment by tbedford** [comment]
@conshus @fabianrbz has fixed those merge conflicts. Thanks Fabian.

**Comment by tbedford** [comment]
@conshus is this ready for me to have another look at again? Thanks.

**Pull request merged**